client/src/components/Pages/Structure/Input/TxSegmentElementInput/TxSegmentElementInput.tsx

```diff
@@ -436,10 +425,18 @@ const TxSegmentCompInput: React.FC<TxSegmentElementInputProps> = ({
     </Box>
   );
 
+  const handleChromosomeChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setTxChrom(e.target.value);
+  };
+
   const genomicCoordinateInfo = (
     <>
       <Box className="mid-inputs">
-        <ChromosomeField fieldValue={txChrom} errorText={txChromText} />
+        <ChromosomeField
+          fieldValue={txChrom}
+          errorText={txChromText}
+          onChange={handleChromosomeChange}
+        />
         <Box mt="18px" width="125px">
           <StrandSwitch setStrand={setTxStrand} selectedStrand={txStrand} />
```

Comment:
Strand can be removed from transcript segment element. Maybe templated sequence too (not sure how this one works)?

Reply:
I'll remove the ability to change it for tx element. I think it might be fine to leave in the UI as a helpful visual tool to the user, though. what are your thoughts on that?

It looks like we still require the user to put in the strand for Templated Sequence, though. That element is pretty much entirely user-made, whereas tx makes a request to cool-seq-tool. If this is something cool-seq-tool can do now, we should def make a new issue for it!

Reply:
pushed a commit where the ability to change it is removed, but it'll still show the strand to the user! I can also just remove it altogether- I just wasn't sure if it was a helpful visual aid or not

Reply:
@katiestahl so it makes sense for transcript segment element when `Genomic coordinates, gene` is selected as input data since strand is automatically generated. However, `Genomic coordinates, transcript` does not change the strand. For example, `NC_000011.10` and `NM_003390.3` should change strand to positive, but it stays negative

Reply:
@korikuzma is it okay if I create a new ticket for that? This is not something that was working or implemented like this before, so I think it's outside of the scope of this ticket. I'll create a new issue though

Reply:
Can we hide the strand for now so it's not confusing and back back in #321

Reply:
yep!